### ISSUE TITLE
No blocks found

### ISSUE THREAD

**Issue by ahrcoin** [sammy007/open-ethereum-pool#270]
#### System information

Geth version: Parity (auto-update) (synced)

Redis version: `redis-server --version`  3.06

OS & Version: Linux/OSX Ubuntu 16.04 LTS

Commit branch and hash: (`master`, `ba60e1c`) Master

#### Expected behaviour
Blocks Found in ~2 days

#### Actual behaviour

No Blocks Found in ~ 1 week

#### Steps to reproduce the behaviour

Pool is running.  Miners are running. Shares are being accepted. but no blocks found that I can find.  Miners show up in redis.  No blocks could be found.  

Just curious if there is a way to see error messages in redis.  Like if for some reason I am finding blocks but they are not being accepted.  Maybe I am just very unlucky.  But calculating it out i should be finding a block ever 1.8 to 2 days.
#### Backtrace

````
[backtrace]
````

**Comment by Serpent6877** [comment]
What coin?

**Comment by kulisu** [comment]
how much hashpower do you have ? 20GH/s ? 40GH/s ? 100GH/s ?

**Comment by techievee** [comment]
The same issue is faced by me, Mininig at 30GHS for 6 days ethereum no block. 

The issue started just 6 days back

Is there any issues are critical  update that need to be done.

Running the setup in Azure Cloud, Is there any chance of this being because of Meltdown and Spectre Vulnerability that is being patched by all cloud providers

Thanks

**Comment by ahrcoin** [comment]
I am set up for music coin and the pool hashrate is at 80mh. Based on the nethash and difficulty it should be like 2 days to find a block.

I know this pool is for ethereum but I am still trying to learn and do not want to get in over my head before I am inowledable of how all this stuff works. It's more of a hobby for me right now so it's just me and a buddy with a couple cards on this project.

let me know if there is any code that may help.

**Comment by ahrcoin** [comment]
Well I think I figured it out.  I will share in case anyone else is having this.

If you are using parity and having miners use stratum mining, there is an option you must put in your config.toml under the mining section for option "stratum".  The stratum feature is disabled by default, I set this as stratum = true and found a block shortly after restarting parity with that enabled.  

The default port is 8008 which is the default of the pool as well so no need to change anything else.

Hope this may help if someone else is struggling with same issue.  This also could have just been coincidence and nothing to do with not finding blocks and maybe my pool was just extremely unlucky.

Closing this issue.
